### PR TITLE
Chapter 3

### PR DESCRIPTION
**Pull request by nxvinh222**
Tạo page tĩnh
Câu hỏi: https://docs.google.com/document/d/1Snv33qyYXbzGBhSEKPV_f0MDknae4iivoBA9QBBl9yk/edit?usp=sharing

### CODE REVIEW

**Comment by nxvinh222** [comment]
image

**Comment by nxvinh222** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+</head>
 
-  <body>
-    <%= yield %>
-  </body>
+<body>
+  <%= link_to "English", locale: :en %>
+  <%= link_to "Vietnamese", locale: :vi %>
+  <%= yield %>
+</body>
```
Comment (ngocvt-0484) [inline app/views/layouts/application.html.erb]: sai indent
e chỉnh lại nhé

**Comment by nxvinh222** [comment]
ready

**Comment by ngocvt-0484** [comment]
commented

**Comment by nxvinh222** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**